src/main/java/org/fundacionjala/coding/william/FizzBuzz.java

```diff
@@ -0,0 +1,39 @@
+package org.fundacionjala.coding.william;
+
+import java.util.StringJoiner;
+import java.util.stream.IntStream;
+
+/**
+ * FizzBuzz class that exchanges the values of digits 3 and 5.
+ */
+public class FizzBuzz {
+
+    private static final int NUMBERTHREE = 3;
+    private static final int NUMBERFIVE = 5;
+    private static final int NUMBERTEN = 10;
+    private static final int NUMBERZERO = 0;
+
+    /**
+     * Method that replaces the values 3 and 5 in a chain.
+     *
+     * @param number is the number where the chain will arrive in numbers.
+     * @return returns a string with the numbers exchanged.
+     */
+    public String fizzBuzz(int number) {
```

Comment:
You can split this method in two
one of them given one number will return fizz, buzz, or fizz buzz